### PR TITLE
[devops] use yarn berry

### PR DESCRIPTION
**Pull request by adrian-gierakowski**
change made while migrating ST to berry

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,14 +38,15 @@
...
+    "aigle": "suguru03/aigle#8739846ba9d4cfc116e1546da1181c73564cae0b",
     "aware": "^0.3.1",
     "bignumber.js": "^9.0.0",
     "eip-712": "1.0.0",
     "lodash": "^4.17.20",
+    "ramda": "^0.27.1",
     "request": "^2.88.2",
     "request-promise": "^4.2.6",
-    "starkware_crypto": "git+https://github.com/DeversiFi/starkware-crypto#504edca95842796908e74dad09518d092fca4e1b",
+    "starkware_crypto": "rhinofi/starkware-crypto#504edca95842796908e74dad09518d092fca4e1b",
```
Comment (piotrfrankowski) [inline package.json:49]: change to scoped

Comment (adrian-gierakowski) [reply package.json]: done

**Review by arijoon — APPROVED** [review]
Would this be considered a breaking change?

**Comment by adrian-gierakowski** [comment]
> Would this be considered a breaking change?

I don't think so as we are not changing any public interfaces

**Pull request merged**